app/models/application_form.rb

```diff
@@ -462,6 +464,10 @@ def qualifications_completed?
       (!science_gcse_needed? || science_gcse_completed)
   end
 
+  def eligible_for_free_school_meals?
```

Comment:
imho `ask_about_free_school_meals?` would be clearer.

incidentally it'd be great to get all of this stuff out of the `ApplicationForm` class, but that's for another day.

Reply:
thanks, i've changed